ansible/roles/setup-localhost/tasks/main.yml

```diff
@@ -12,17 +12,17 @@
 - name: wait 300 seconds for port 22 to become open and contain "OpenSSH"
   ansible.builtin.wait_for:
     port: 22
-    host: "{{ item }}"
+    host: "{{ ansible_host }}"
```

Comment:
Hardcoding all these hosts to `ansible_host` would break the loop `with_inventory_hostnames`